arangod/RocksDBEngine/RocksDBBuilderIndex.cpp

```diff
@@ -303,6 +304,15 @@ arangodb::Result RocksDBBuilderIndex::fillIndexForeground() {
     _docsProcessed.fetch_add(docsProcessed, std::memory_order_relaxed);
   };
 
+  // reserve some space in WriteBatch
+  size_t batchSize = 1024 * 1024;
```

Comment:
maybe this could be a constant  cariable in the .h?

Reply:
I don't think it should be exposed in the .h file. It is internal to the index builder code, and the value does not need to be accessed from other places.